### PR TITLE
update `role` for quick input element

### LINKED ISSUE #203493: Incorrect role announced by screen reader on string parameter input box of a command in command center
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.85.2
- OS Version: Windows 11 Enterprise (Windows_NT x64 10.0.22631.3007)

```
Version: 1.85.2 (user setup)
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:10.514Z
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
```

**Description**
We are building a VS Code extension. During the accessibility test, the tester reported that Narrator says "Combo edit expanded" on a pure text input of a string command parameter of a command in command center.

I go ahead and checked the role of the input box and I notice that the role is a "combobox". I feel it's improper to call a pure text input with helper text a combobox since it does not provide any options to choose from.
image
image

Can we check if the role is intentionally set correctly?
Thanks.

### PR DESCRIPTION
**Pull request by meganrogge**
fix #203493

### CODE REVIEW

File: src/vs/platform/quickinput/browser/quickInputBox.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class QuickInputBox extends Disposable {
 		this.container = dom.append(this.parent, $('.quick-input-box'));
 		this.findInput = this._register(new FindInput(this.container, undefined, { label: '', inputBoxStyles, toggleStyles }));
 		const input = this.findInput.inputBox.inputElement;
-		input.role = 'combobox';
+		input.role = 'input';
```
Comment (TylerLeonhardt) [inline src/vs/platform/quickinput/browser/quickInputBox.ts:32]: Are you sure this is right? I would think that for quickpicks (which are about choosing an option) are more like a combo box than an input.

Comment (TylerLeonhardt) [reply src/vs/platform/quickinput/browser/quickInputBox.ts:32]: maybe this should be set to input when it's an input box not a quick pick

Comment (meganrogge) [reply src/vs/platform/quickinput/browser/quickInputBox.ts:32]: This element is the actual input box part of the quickpick I think? Nothing can be chosen from within it - a user navigates to the list of items to do so.

Comment (meganrogge) [reply src/vs/platform/quickinput/browser/quickInputBox.ts:32]: I will defer to you as you own this area. If you think `combobox` is most appropriate, feel free to close the issue with that explanation.

Comment (TylerLeonhardt) [reply src/vs/platform/quickinput/browser/quickInputBox.ts:32]: >  a user navigates to the list of items to do so.

The user can do it this way by using TAB, but the default experience is that the input has Dom focus and key handlers attached to the input allow for choosing and selecting of the item. A close comparison would be this combobox example:
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/

**Pull request closed without merging**